lib/capn-malloc.c

```diff
@@ -25,6 +25,10 @@ struct check_segment_alignment {
 };
 
 static struct capn_segment *create(void *u, uint32_t id, int sz) {
+	// Silence warnings about unused parameters.
```

Comment:
TBH, this is pretty ugly; the better fix is to use `-Wno-unused-parameters`.  In any C project of nontrivial size, there will be cases where you're implementing a function to match a signature (e.g. callback pointer, like here) and some parameters will be unused. There are many useful warnings, but this is not one of them.